sunpy/tests/tests/test_self_test.py

```diff
@@ -100,38 +83,6 @@ def test_main_figure_dir(monkeypatch):
                     [root_dir, '--figure_dir', '.', '-m', 'figure'])
 
 
-def test_main_coverage(monkeypatch):
```

Comment:
I removed coverage since we don't actually support it normally, only via the CI.